yarn-project/blob-sink/src/types/api.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+
+export interface PostBlobSidecarRequest {
+  // eslint-disable-next-line camelcase
+  block_id: string;
+  blobs: Array<{
+    index: number;
+    blob: {
+      type: string;
+      data: string;
+    };
+  }>;
+}
+
+// Validation schemas
+export const blockIdSchema = z.coerce
```

Comment:
Block id here not the same as https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getBlobSidecars
Need to be more explicit about it.